packages/ra-ui-materialui/src/layout/CardContentInner.js

```diff
@@ -4,18 +4,22 @@ import classnames from 'classnames';
 import CardContent from '@material-ui/core/CardContent';
 import { withStyles } from '@material-ui/core/styles';
 
-const styles = {
-    root: {
-        paddingTop: 0,
-        paddingBottom: 0,
-        '&:first-child': {
-            paddingTop: 16,
-        },
-        '&:last-child': {
-            paddingBottom: 16,
-        },
+var styles = theme => ({
+  root: {
+    paddingTop: 0,
+    paddingBottom: 0,
+    '&:first-child': {
+      paddingTop: 16,
     },
-};
+    '&:last-child': {
+      paddingBottom: 16,
+      // Reset on mouse devices
```

Comment:
Can you remove this comment? It does not bring anything as the code is clear enough

Reply:
done

Reply:
I'd still like a screenshot please :)